group_project_v2/public/js/project_navigator/ask_ta_view.js

```diff
@@ -0,0 +1,32 @@
+function GroupProjectNavigatorAskTAView(runtime, element) {
+    var form = $(".contact-ta-form", element),
+        textarea = $("textarea", form);
+
+    $(textarea).on('keyup', function () {
```

Comment:
@Kelketek there wouldn't be any text, at least in Chrome. This is a pre-existing behavior anyway.